### ISSUE TITLE
Error with import_medications ()

### ISSUE THREAD

**Issue by zenourn** [nzbri/chchpd#8]
When running import_medications I get an error:

```
medications = import_medications()
Error: Assigned data `0` must be compatible with existing data.
Error occurred for column `other_medications`.
x No common type for `value` <double> and `x` <character>.
```

I assume this is likely a data problem. There are now sheets 'PD Progression Clinical Data' and 'NEW PD Progression Clinical Data'. Has this function been updated to use NEW? Are there any changes that would of caused this?

**Comment by m-macaskill** [comment]
This is working for me currently. Can you check that it is still an issue for you?

Maybe an erroneous data cell has been corrected. But even so, I'm not sure how this error could arise at the import stage, as columns have to be explicitly coerced into a particular data type in `googlesheets4` (previously, it would try to do an informed guess, with only some columns manually overridden by `chchpd`).

Yes, this function reads from the new clinical spreadsheet ( a common file reference ID is used by all functions that access that spreadsheet).

**Comment by zenourn** [comment]
Yes, it is still happening, so that points that it isn't a data error. The full code I'm running is:
```
googlesheets::gs_auth(key = "xxx", 
                      secret = "yyy")

devtools::install_github('nzbri/chchpd')
library(chchpd)
library(dplyr)

participants = import_participants()
np = import_neuropsyc(concise=FALSE)
updrs = import_motor_scores()
mri = import_MRI()
medications = import_medications()
```

Could you give me your `sessionInfo()` so I can see what could potentially cause this? My versions are:

```
> sessionInfo()
R version 3.6.1 (2019-07-05)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS Catalina 10.15.3

Matrix products: default
BLAS:   /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRlapack.dylib

locale:
[1] en_NZ.UTF-8/en_NZ.UTF-8/en_NZ.UTF-8/C/en_NZ.UTF-8/en_NZ.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] dplyr_0.8.5  chchpd_0.3.0

loaded via a namespace (and not attached):
 [1] tidyselect_1.0.0    xfun_0.13           janitor_2.0.1       remotes_2.1.1      
 [5] purrr_0.3.3         gargle_0.4.0        snakecase_0.11.0    vctrs_0.2.4        
 [9] generics_0.0.2      testthat_2.3.2      usethis_1.5.1       rlang_0.4.5        
[13] pkgbuild_1.0.6      pillar_1.4.3        glue_1.4.0          withr_2.1.2        
[17] sessioninfo_1.1.1   lifecycle_0.2.0     stringr_1.4.0       googlesheets_0.3.0 
[21] cellranger_1.1.0    devtools_2.2.2      memoise_1.1.0       knitr_1.28         
[25] callr_3.4.3         ps_1.3.2            curl_4.3            fansi_0.4.1        
[29] highr_0.8           Rcpp_1.0.4.6        readr_1.3.1         openssl_1.4.1      
[33] backports_1.1.5     googlesheets4_0.1.1 desc_1.2.0          pkgload_1.0.2      
[37] jsonlite_1.6.1      fs_1.4.1            askpass_1.1         hms_0.5.3          
[41] packrat_0.5.0       digest_0.6.25       stringi_1.4.6       processx_3.4.2     
[45] rprojroot_1.3-2     cli_2.0.2           tools_3.6.1         magrittr_1.5       
[49] tibble_3.0.0        crayon_1.3.4        tidyr_1.0.2         pkgconfig_2.0.3    
[53] ellipsis_0.3.0      prettyunits_1.1.1   lubridate_1.7.8     assertthat_0.2.1   
[57] httr_1.4.1          rstudioapi_0.11     R6_2.4.1            compiler_3.6.1  
```

**Comment by rezashr** [comment]
I think this is related to the version of CHCHPD package. Maybe update to the latest version, which uses googlesheets4 and the new clinical spreadsheet. Although it seems that you are using the version 0.3.0. I just ran your code and it works fine for me too.

**Comment by zenourn** [comment]
I've run `devtools::install_github('nzbri/chchpd')` a couple of times now (second time with force=TRUE) which hopefully means I do have the latest version?

I think googlesheets4 is in use (it popped up a page asking me to authenticate) that old key/secret isn't used and commenting it out everything still worked fine (except for the medications import error). 

I'll try on my Windows desktop and see what happens.

**Comment by m-macaskill** [comment]
The issue is the `tibble` package. I was using `tibble_2.1.3`. I upgraded to `tibble_3.0.0` to match you and now I get the error as well. Think I can see where the error arises now.

**Comment by m-macaskill** [comment]
Thanks for reporting. Should be fixed now if you update `chchpd` to 0.3.1.
